app/components/dashboard/run/left-panel/component.js

```diff
@@ -165,5 +166,63 @@ export default Component.extend(FullScreenMixin, {
         denyDelete() {
             return true;
         },
+        
+        exportTale(id) {
+          let self = this;
+          // Asks the girder endpoint for a zipped Tale
+          // This code was adapted from
+          // https://stackoverflow.com/questions/16086162/handle-file-download-from-ajax-post
+          const token = self.get('tokenHandler').getWholeTaleAuthToken();
+          let url = `${config.apiUrl}/tale/${id}/export`;
+          var xhr = new XMLHttpRequest();
+          xhr.open('GET', url, true);
+          xhr.responseType = 'arraybuffer';
+          xhr.setRequestHeader("Girder-Token", token);
+          self.set('showDimmer', true);
+          xhr.onload = function () {
+              if (this.status === 200) {
+                self.set('showDimmer', false);
+                  var filename = "";
+                  var disposition = xhr.getResponseHeader('Content-Disposition');
+                  if (disposition && disposition.indexOf('attachment') !== -1) {
+                      var filenameRegex = /filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/;
+                      var matches = filenameRegex.exec(disposition);
+                      if (matches != null && matches[1]) filename = matches[1].replace(/['"]/g, '');
+                  }
+                  var type = xhr.getResponseHeader('Content-Type');
+    
+                  var blob = typeof File === 'function'
+                      ? new File([this.response], filename, { type: type })
+                      : new Blob([this.response], { type: type });
+                  if (typeof window.navigator.msSaveBlob !== 'undefined') {
+                      // IE workaround for "HTML7007: One or more blob URLs were revoked by closing the blob for which they were created. These URLs will no longer resolve as the data backing the URL has been freed."
+                      window.navigator.msSaveBlob(blob, filename);
```

Comment:
No need for IE workarounds anymore imo, since Microsoft doesn't even consider it a browser.